astroid/inference_tip.py

```diff
@@ -3,16 +3,20 @@
 
 """Transform utilities (filters and decorator)"""
 
-import typing
+
+from typing import Any, Callable, Dict, Iterator, List, Optional, Tuple, Type, Union
```

Comment:
It's imported this way in astroid because List, Dict, and Tuple are also astroid.nodes so it can become confusing. It's probably worth it to stay consistent accross the code base.

Reply:
In this file we only use 2 nodes and a lot of different typing stuff. That's why I opted to do it the other way around in this file.

I'm fine with reverting!

Reply:
Reverted to only import `typing`.